libs/product/src/models/product.ts

```diff
@@ -1,11 +1,17 @@
 import { DaffProductImage } from './product-image';
 
+export enum DaffProductTypeEnum {
+	SimpleProduct = 'Simple Product',
+	BundledProduct = 'Bundled Product',
+	ConfigurableProduct = 'Configurable Product'
+}
+
 /**
  * An interface for a product usable by the @daffodil/product library.
  */
 export interface DaffProduct {
 	id: string;
-	__typename?: string;
+	__typename?: DaffProductTypeEnum;
```

Comment:
`type` instead of `__typename`

Reply:
I wouldn't say we depend on GraphQl by calling it __typename. We still transform from the magento object to a daffodil one. We don't spread. I just think it'll feel more familiar to people who know graphQL. Why change it for no reason?